### DISCUSSION TITLE
MCU::LCDDisplay is showing pan value instead of track names

### DISCUSSION THREAD

**Discussion by ACuppaJoes** [tttapa/Control-Surface#692] [Q&A]
Hey,

I am using the [MCU-OLED-SSD1306-x2](https://tttapa.github.io/Control-Surface-doc/Doxygen/d0/d84/MCU-OLED-SSD1306-x2_8ino-example.html#_a8) example but for some reason, MCU::LCDDisplay is showing pan values instead of track names. I'm not sure how to fix this but I know that this is definitely incorrect. The [example video](https://www.youtube.com/watch?v=upL609Vkzug&feature=emb_title) does seem to show track names in the same place.

Any help would be great!

**Comment by tttapa** [comment]
The `MCU::LCDDisplay` class just displays whatever it receives over MIDI, so this is probably a setting in the DAW.  
IIRC, there are different modes for the MCU, you might want to experiment with some of the `VIEW` constants here: https://tttapa.github.io/Control-Surface-doc/Doxygen/de/d44/group__MCU__Notes.html
